app/models/supervisor.rb

```diff
@@ -0,0 +1,26 @@
+require 'yaml'
+
+class Supervisor
+
+  YAML_READER = lambda {|file, id| YAML::load(IO.read(file)).symbolize_keys.merge({id: id})}
+
+  def self.get_maniphests
```

Comment:
maniphest -> manifest
nie ma takiego słowa w angielskim jak manifest, a rozumiem, że chodziło o coś w rodzaju: http://www.w3.org/TR/appmanifest/

to tyczy się też ścieżek i innych wystąpień